ifopt-config.cmake

```diff
@@ -35,6 +35,7 @@
 #
 #=============================================================================
 
+find_dependency(Eigen3 REQUIRED)
```

Comment:
It is my understanding that REQUIRED can be removed

Reply:
See [here](https://cmake.org/cmake/help/latest/module/CMakeFindDependencyMacro.html) it should forward what was used when calling find_package.

Reply:
i've removed REQUIRED